procedures/Transitioning.rst

```diff
@@ -80,77 +59,21 @@ which require that status need to be check and revoked. (Refer to the
     +----+-----+----+------------------------------------------------------------------------------------------------------------------------+
     | X  | X   |    | Release google group                                                                                                   |
     +----+-----+----+------------------------------------------------------------------------------------------------------------------------+
-    | X  | X   |    | Release Trello group                                                                                                   |
-    +----+-----+----+------------------------------------------------------------------------------------------------------------------------+
-    | X  |     |    | other Google Drive documents [#gd_docs]_                                                                               |
-    +----+-----+----+------------------------------------------------------------------------------------------------------------------------+
     | X  |     |    | re-assign bugs                                                                                                         |
     +----+-----+----+------------------------------------------------------------------------------------------------------------------------+
-    | X  |     |    | named contact in Nagios escalation chain                                                                               |
-    +----+-----+----+------------------------------------------------------------------------------------------------------------------------+
-    | X  |     |    | Allowed to merge to puppet production [#special]_                                                                      |
-    +----+-----+----+------------------------------------------------------------------------------------------------------------------------+
-
-Detailed Instructions
----------------------
-.. _password_list:
 
 Password List
 ^^^^^^^^^^^^^
 
-.. [#passwords]
+.. [#secrets]
 
-    Almost all passwords are now shared via gpg encrypted files. To get a
-    list of passwords shared with a user, use the
-    "``scripts/who_knows_what``" script to identify passwords that
-    should be changed.
-    Also, add the
-    departing user's id to the "``scripts/alumni.json``" file.
-
-    .. note:: access to the repo requires both membership in both the
-      ``vpn_git_internal`` (or ``vpn_releng``) and either (``relops`` or
-      ``releng``).
-
-.. [#mozbuild]
-
-    To change the password on an IRC channel where you have ops
-    permissions:
-
-        - Make sure user is *not* in #mozbuild, a kick or ban is
-          sometimes necessary (due to auto-reconnect)
-        - ``/msg chanserv set #mozbuild mlock +k newpass``
+    RelEng passwords and other credentials are stored in a `SOPS <https://github.com/mozilla/sops>` repository.
```

Comment:
Consider another sentence or two here saying what needs to be done in SOPS?

Reply:
I'm not sure if we're allowed to -- @MihaiTabara - do you know?